packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -21,9 +21,14 @@ Parameters:
     Default: ''
     Type: String
   FileAssetsBucketKmsKeyId:
-    Description: Empty to create a new key (default), 'AWS_MANAGED_KEY' to use a managed
-      S3 key, or the ID/ARN of an existing key.
-    Default: ''
+    Description: "'DEFAULT_ENCRYPTION' to use a managed S3 key, 'BOOTSTRAP_KEY' to use bootstrap CMK key (default),
+      or the ID/ARN of an existing key."
+    Default: 'BOOTSTRAP_KEY'
```

Comment:
We can't change the default values here, I believe that would be a breaking change.

Reply:
It should be fine to change them as long as we also change the inputs. Like we did here https://github.com/aws/aws-cdk/blob/8432fd2077e09e74d9cc91c506193130fa1475c3/packages/aws-cdk/lib/api/bootstrap/bootstrap-environment.ts#L194-L199

So in the end the conditions and everything around them does not change